ubuntudesign/www.ubuntu.com
# Port helpers.js to polyfills.js

## Done
- Removed all the helper functions from helpers
- Renamed helpers.js to polyfills.js as it only contains a polyfill
- Removed the A/B test on the contribution page

## QA
- Load the site and check there are no js errors
- Go to `http://localhost:8001/download/desktop/contribute?version=16.04.1&architecture=amd64` and check the title loads ok and there are no errors in the console
- Check for errors on: `/cloud/openstack/managed-cloud/` as the helpers import was removed
